### PR TITLE
Styling/scroll bar

### PR DESCRIPTION
**Pull request by attom2**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the new behavior?

- Adds react-select npm package to style a drop down menu
- Puts front and back image of pokemon on pokedex
- Makes pokemon shiny if up arrow is pressed
- displays random pokemon if down arrow is pressed

## Does this introduce a breaking change?

- [x] Yes
- [ ] No

Tests will break from how we select pokemon

## Other information

image
image

### CODE REVIEW

File: src/Components/PokeDetails/PokeDetails.js [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
-  const pokemonName = details.name
-  const pokeFrontImage = details.sprites.front_default
+const PokeDetails = ({details, isShiny}) => {
+  const pokemonName = details.name;
+  const pokemonType = details.types.map(poke => poke.type.name + '\n');
+  const pokeFrontImage = details.sprites.front_default;
+  const pokeBackImage = details.sprites.back_default;
+  const pokeBackShiny = details.sprites.back_shiny;
+  const pokeFrontShiny = details.sprites.front_shiny;
+  console.log(details);
```
Comment (emontealvo) [inline src/Components/PokeDetails/PokeDetails.js:11]: could we de-structure some of these variables, or at least the sprites.

File: src/Components/Pokedex/Pokedex.js [hunk 2]
```diff
@@ -1,50 +1,54 @@
...
 
   return (
     <section className="pokedex-outline">
       <section className="pokedex-screen">
-        {createSelectMenu()}
-        {singlePokemon.name && <PokeDetails details={singlePokemon}/> }
+        <Select
+          onChange={displaySinglePokemon}
+          options={createPokeList()}
+        />
```
Comment (emontealvo) [inline src/Components/Pokedex/Pokedex.js:38]: That's a pretty crafty package! We'll have to talk more about inner-workings.

**Comment by emontealvo** [comment]
Great work! We can probably bring over the mix-in for the background in pokedex into the select menu background if desired.

File: src/Components/PokeDetails/PokeDetails.js (hunk 1 above)
Comment (attom2) [reply src/Components/PokeDetails/PokeDetails.js]: done!

File: src/Components/Pokedex/Pokedex.js (hunk 2 above)
Comment (attom2) [reply src/Components/Pokedex/Pokedex.js]: lots of goodies in that package

**Comment by MarchCorbin** [comment]
Really cool design for the list items!

File: src/Components/PokeDetails/PokeDetails.js [hunk 3]
```diff
@@ -1,23 +1,32 @@
...
-      />
+      <section className="img-container">
+        <img
+          className='pokemon-pic'
+          src={isShiny ? front_shiny : front_default}
+          alt={name}
+        />
+        <img
+          className='pokemon-pic'
+          src={isShiny ? back_shiny : back_default}
```
Comment (MarchCorbin) [inline src/Components/PokeDetails/PokeDetails.js:18]: A shiny pokemon is a happy one.

**Pull request merged**